src/index.html

```diff
@@ -16,16 +17,19 @@
       href="style.css"
     />
   </head>
+  <!-- #endregion -->
   <body>
     <input
+      class="bar big-bar"
       type="text"
-      data-qa="keypress"
+      data-qa="big keypress"
       placeholder="Try “Los Angeles“"
     />
 
     <input
+      class="bar small-bar"
       type="text"
-      data-qa="hover"
+      data-qa="small"
       placeholder="Try “Los Angeles“"
     />
```

Comment:
Similarly, consider adding a `<label>` element for this input field to improve accessibility for screen readers.